INSTALL.md

```diff
@@ -0,0 +1,41 @@
+# Getting started
+
+The `iroh-blobs` protocol is meant to be use in conjunction with `iroh`. [Iroh](https://docs.rs/iroh/latest/iroh/index.html) is a networking library for making direct connections, these connections are what power the data transfers in `iroh-blobs`.
```

Comment:
usually best to link to `docs.rs/iroh`, the redirects will take care of the rest